orc8r/cloud/helm/orc8r/values.yaml

```diff
@@ -79,7 +79,7 @@ nginx:
       clientcertLegacy: {}
       proxy: {}
     labels: {}
-    type: ClusterIP
+    type: LoadBalancer
```

Comment:
Same as my below question... maybe this is just an incorrect default that always gets overridden? But either way, yes seems like this should indeed be denoted as LB type

Reply:
Same https://github.com/magma/magma/pull/6413#discussion_r740730205